mlir/utils/buildbot/build-run.sh

```diff
@@ -71,4 +71,4 @@ fi
 docker build ${BUILDPROXY} -t "${IMAGE_NAME}:latest" .
 # Use "--group-add root" to work around container putting /dev/kfd and /dev/dri
 # in root instead of the proper video or render.
-docker run -d --net=host --device=/dev/kfd --device=/dev/dri --group-add video --group-add root --name buildbot --restart always ${RUNPROXY} ${VOLUMES} ${ARGS} "${IMAGE_NAME}:latest" ${CMD}
+docker run -d --net=host --device=/dev/kfd --device=/dev/dri --group-add video --group-add render --group-add root --name buildbot --restart always ${RUNPROXY} ${VOLUMES} ${ARGS} "${IMAGE_NAME}:latest" ${CMD}
```

Comment:
Somehow forgot the render group, for months, and got away with it until recently.